### PR TITLE
OVF Import: data disk steps to use workflow default timeout

### PR DESCRIPTION
**Pull request by zoran15**
OVF Import: data disk steps to use workflow default timeout instead of timeout being hardcoded to 1hr

### CODE REVIEW

File: daisy/step.go [hunk 1]
```diff
@@ -69,8 +69,12 @@ type Step struct {
 	testType stepImpl
 }
 
-// NewStep creates a Step with given name and timeout  with the specified workflow
+// NewStep creates a Step with given name and timeout  with the specified workflow.
```
Comment (dntczdx) [inline daisy/step.go]: I would create a new method (NewStepWithDefaultTimeout?) for this purpose because timeout<=0 looks like a hidden logic

Comment (EricEdens) [reply daisy/step.go]: +1

File: daisy/step.go [hunk 2]
```diff
@@ -69,11 +69,20 @@ type Step struct {
...
+// If timeout is less or equal to zero, defaultTimeout from the workflow will be used
 func NewStep(name string, w *Workflow, timeout time.Duration) *Step {
+	if timeout <= 0 {
+		return &Step{name: name, w: w, Timeout: w.DefaultTimeout}
+	}
 	return &Step{name: name, w: w, timeout: timeout}
 }
 
+// NewStepDefaultTimeout creates a Step with given name using default timeout from the workflow
+func NewStepDefaultTimeout(name string, w *Workflow, timeout time.Duration) *Step {
```
Comment (EricEdens) [inline daisy/step.go]: need to remove the timeout param?

Comment (zoran15) [reply daisy/step.go]: Yeah. I think I'll go home get some sleep. :)

**Pull request merged**